db/migrate/20231104023717_create_posts.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.integer :author_id
```

Comment:
- [ ] The author_id you have here should be referenced and not passed in as an integer. below is an example of what is expected

E.g
```ruby
   create_table :posts do |t|
    t.string :title
    t.text :text
    t.integer :comments_counter
    t.integer :likes_counter
    t.references :author, foreign_key: { to_table: 'users' }
    t.timestamps 
  end
```

N.B: remember to run the `db:migrate` to have your schema file updated from the changes you made in your migration file